Group dependencies for submodules (#67)
facebookresearch/AugLy: augly/video/README.md

# Video

## Installation

> If you plan to use the video augmentations from AugLy, you can install using the following command:

Reviewer: ```suggestion
If you would like to use the video augmentations, please install AugLy using the following command:
```
